src/components/header/styleHeader/loginButton.scss

```diff
@@ -0,0 +1,17 @@
+button {
+    background: none;
+    color: rgb(241, 160, 8);
```

Comment:
This color is set as a primary color inside in header.scss. Please move it in one place and use it all over the app.